tasties_app/templates/tasties_app/single_recipe.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'tasties_app/base.html' %}
+{% load static %}
+{% block css %}
+    {% load bootstrap_icons %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/single_recipe.css' %}" />
+{% endblock %}
+{% block content %}
+    <div class="recipe-body">
+        <div class="recipe-header">
+            <h1>{{ recipe.title }}</h1>
+            <h2>{{ recipe.description }}</h2>
+            <div class="recipe-header-info">
+                <span class="author-name">by @{{ recipe.author_id }}</span>
+                <span class="recipe-date">{{ recipe.publication_date }}</span>
+            </div>
+        </div>
+
+        <div class="categories">
+            {% for category in categories %}
+                <button class="btn btn-light" name="{{ category.category_name }}">{{ category.category_name }}</button>
+            {% endfor %}
+
+        </div>
+
+        <div class="img-and-ingredients">
+            <img src="{% static recipe.recipe_picture %}" alt="{{ recipe.title }}" class="recipe-image">
+            <div class="container">
+                <div class="time-and-rating">
+                    <div class="recipe-time">
+                        <span class="recipe-time-icon">{% bs_icon 'clock-history' %}</span>
+                        <span class="recipe-time-text">{{ recipe.minutes_to_make }} mins</span>
+                    </div>
+                    <div class="recipe-rating">
+                        <span class="recipe-rating-icon">{% bs_icon 'star-fill' %}</span>
+                        <span class="recipe-rating-text">{{ rating }}</span>
+                    </div>
+                </div>
+            <hr>
+                <ul class="ingredients-list list-group list-group">
+                    <h3>Ingredients</h3>
+                    {% for ingredient in ingredients %}
+                        <li class="ingredient">
+                            {{ ingredient }}
```

Comment:
Here you're just calling the `Ingredient` model's `__str__` method; This is fine for the time being, but please notice
that we will need to modify/augment this in the future. For example, I think that we should (in the future) omit the word "Whole" from the string, should it be the measurement unit - think "5 Chicken breasts Whole" etc. We would also like to do unit conversion, and that will have to happen somewhere.